src/Mod/Sketcher/Gui/CommandSketcherVirtualSpace.cpp

```diff
@@ -74,13 +74,14 @@ bool isSketcherVirtualSpaceActive(Gui::Document* doc, bool actsOnSelection)
 
 void ActivateVirtualSpaceHandler(Gui::Document* doc, DrawSketchHandler* handler)
 {
+    std::unique_ptr<DrawSketchHandler> ptr(handler);
```

Comment:
Why make this a local instead changing the signature like you did with ActivateHandler?